docs/vectorizer-quick-start.md

```diff
@@ -0,0 +1,130 @@
+# Vectorizer quick start
+
+This page shows you how to create a vectorizer in a self-hosted Postgres instance, then use 
+a pgai vectorizer worker to create embeddings from data in your database. You view the 
+embeddings with a standard SQL query. 
+
+## Prerequisites
+
+To follow the steps in this page, you need to:
+- [Create a local developer environment](/docs/vectorizer-worker.md#install-the-timescaledb-ha-docker-image)
```

Comment:
Not a big fan of making people jump around different pages so much. I liked it more when the compose was in here. I also think the compose should not be used for anything else, its way too minimal for production use so having it in a more general file is just confusing.

Reply:
I am in 2 minds. I agree that having all in one place for a quickstart is easier. However, you get different instructions and then there is no consistent way of working. For example, the docker image is the standard Timescale dev environment, and the install guide is here: https://docs.timescale.com/self-hosted/latest/install/installation-docker/. 

In terms of this repo I would prefer a single source of truth and have all install guides here: https://github.com/timescale/pgai/blob/vectorizer-naming-update/README.md#installation.

Reply:
I've put the dev env back in the quickstart. The words didn't quite work in the run page.